flow/project.py

```diff
@@ -2848,35 +2855,50 @@ class MyProject(FlowProject):
         # Set verbosity level according to the `-v` argument.
         logging.basicConfig(level=max(0, logging.WARNING - 10 * args.verbose))
 
-        def _exit_or_raise():
+        def _show_tb_and_exit(error):
             if args.show_traceback:
-                raise
```

Comment:
In my understanding, the purpose of this `raise` was just to show a traceback -- I assume that we don't actually need to raise, since this is a `main()` function.

Reply:
Agreed.